task3/update/index.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="logo.png">
+    <title> Ribbitter | Feed </title>
+</head>
+<body>
+<header>
+    <nav class="container">
+        <a class="logo" href="index.html">
+            <img src="logo.png" alt="A VERY MUCH GREEN FROG">
+            <span class="logotext"><h1>Ribbitter</h1></span>
+        </a>
+        <a class="write" href="">
+            <img src="edit.png" alt="Write new post">
+        </a>
+        <a class="exit" href="">
+            <img src="exit.png" alt="exit">
+        </a>
+        <a class="user" href="">
+            <span class="yourname">YourName</span>
+            <img src="you.png" alt="your photo">
+        </a>
+    </nav>
+</header>
+<aside>
+    <h2 class="prefs">Your content preferences:</h2>
```

Comment:
1) class name should describe position or purpose/effect that css class provieds.
2) h2 - mean the title for the second heading, if site were a book, it would be a name of chapter, so does the preferences is something that user searching for on your site?